src/models/stores/document-group.test.ts

```diff
@@ -1,15 +1,15 @@
-import { IObservableArray, observable } from "mobx";
+import { observable } from "mobx";
 import { createDocumentModel, DocumentModelSnapshotType, DocumentModelType } from "../document/document";
 import { DocumentContentSnapshotType } from "../document/document-content";
 import { ProblemDocument } from '../document/document-types';
 import { ClassModel, ClassModelType, ClassUserModel } from "./class";
 import { GroupModel, GroupsModel, GroupsModelType, GroupUserModel } from "./groups";
 import { DeepPartial } from "utility-types";
-import { IDocumentMetadata } from "../../../shared/shared";
-import { ISortedDocumentsStores, SortedDocuments } from "./sorted-documents";
+import { ISortedDocumentsStores, MetadataDocMapModel, SortedDocuments } from "./sorted-documents";
 import { DB } from "../../lib/db";
 import { mock } from "ts-jest-mocker";
 import { Bookmark, Bookmarks } from "./bookmarks";
+import { SnapshotIn } from "mobx-state-tree";
```

Comment:
Not super important, but imports from node modules are being "mixed in" with imports from internal files here which doesn't follow our usual pattern.